### PR TITLE
Add Puyo Puyo Tetris 2 Support

### PR DESCRIPTION
**Pull request by Sajidur78**

### CODE REVIEW

File: HedgeModManager/Game.cs [hunk 1]
```diff
@@ -68,11 +68,29 @@ public static class Games
...
         };
 
+        public static Game Tenpex = new Game()
+        {
+            GameName = "Puyo Puyo Tetris 2",
+            ExecuteableName = "PuyoPuyoTetris2.exe",
+            HasCustomLoader = true,
+            SupportsCPKREDIR = false,
+            ModLoaderDownloadURL = Resources.URL_TML_DL,
+            ModLoaderData = null,
```
Comment (RagdollClash) [inline HedgeModManager/Game.cs:78]: are you not gonna do an embedded loader for PPT2 in this PR?

Comment (Sajidur78) [reply HedgeModManager/Game.cs:78]: we're not going to have an embedded loader for PPT2, the exe becomes really big with it.

Comment (RagdollClash) [reply HedgeModManager/Game.cs:78]: ah

**Review by RagdollClash — APPROVED** [review]
2021-03-24_03-22-59_PuyoPuyoTetris2

**Comment by Sajidur78** [comment]
LGTM

**Review by RagdollClash — COMMENTED** [review]
doesn't LGTM, loser

2021-03-23_15-25-25_PuyoPuyoTetris2

**Pull request merged**